tictactoe.Tests/GameTest.cs

```diff
@@ -143,7 +164,7 @@ public void GamePlayChecksForDraw()
                     "X", "O", "X",
                     "X", "O", "O",
                     "O", "X", "X"};
```

Comment:
Given that you're simulating user input below, do we still need to set up the board?

Reply:
we shouldnt need the board for this test. commit e9f1405f075b6817c23e1f3c35943e7be778cd81